db/migrate/20221101203546_alter_targer_table.rb

```diff
@@ -1,7 +1,7 @@
 class AlterTargerTable < ActiveRecord::Migration[7.0]
   def change
-    remove_column :targets, :latitude
-    remove_column :targets, :longitude
-    add_column :targets, :location, :point, geographic: true
+    remove_column :targets, :latitude, :float
```

Comment:
It's not recommended to edit migrations. This can cause some issues as you need to re-run the migration and you could lose data by running some rollbacks

Reply:
Adding what @jpascual1994 said, if you need to change something like that you can create a new migration updating colum data type.